### PR TITLE
Coder.Coder version 0.11.0

### PR DESCRIPTION
**Pull request by matifali**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/86416)

### CODE REVIEW

**Comment by matifali** [comment]
@microsoft-github-policy-service agree [company="{Coder Technologies, Inc.}"]

**Comment by matifali** [comment]
@microsoft-github-policy-service agree company="Coder Technologies, Inc."

File: manifests/c/Coder/Coder/0.11.0/Coder.Coder.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/coder/coder/releases/download/v0.11.0/coder_0.11.0_windows_amd64_installer.exe
+  InstallerSha256: E50BCB018AE80F14D5A0263D4124FD97FE6501F1D8FFCA31F8CE34BED81928FE
+  UpgradeBehavior: install
+  Commands:
+  - coder
+  ReleaseDate: 2022-10-25
```
Comment (ItzLevvie) [inline manifests/c/Coder/Coder/0.11.0/Coder.Coder.installer.yaml:15]:
```suggestion
  ReleaseDate: 2022-10-25
  AppsAndFeaturesEntries:
  - DisplayVersion: 0.11.0.202210251503
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Duplicate of PR 86238 - to add changes please update the now merged manifest;

**Pull request closed without merging**